woocommerce/woocommerce-ios
# Fixed TitleAndEditableValueTableViewCell constraints errors

Fixes #2259

## Changes
- Fixed constraints in `TitleAndEditableValueTableViewCell` that now use a vertical UIStackView.
- Renamed the `EditableValueOneTableViewCellTests` file, which containts the TitleAndEditableValueTableViewCell tests.

New setup:
<img width="381" alt="Schermata 2020-05-20 alle 19 28 47" src="https://user-images.githubusercontent.com/495617/82478363-1dfa3280-9ad1-11ea-9412-82e831d76f27.png">

## Testing
1) Navigate to an order detail, and try to add a shipping tracking.
2) Make sure that the appearance is like before.

<img src="https://user-images.githubusercontent.com/495617/82478474-3ff3b500-9ad1-11ea-9f0e-e28a915a8a79.png" width=300 />


Update release notes:
